src/app/containers/FrontPageMain/index.test.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { render, cleanup } from '@testing-library/react';
 import { matchSnapshotAsync } from '@bbc/psammead-test-helpers';
 import FrontPageMain from '.';
-import frontPageDataPidgin from '#data/pidgin/frontpage';
+import frontPageDataPidgin from '#data/pidgin/frontpage/index-light';
```

Comment:
Do you think it would be beneficial to add a comment explaining what `index-light` data is?

Reply:
Yes, good suggestion. Please see the latest commit.